source/documentation/deploying_services/sqs.md

```diff
@@ -0,0 +1,342 @@
+
+- `standard` plans configure queues that provide at-least-once delivery,
+which means that each message is delivered at least once.
+- `fifo` (first-in-first-out) plans configure queues that provide [exactly-once
+processing](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html#FIFO-queues-exactly-once-processing),
+which means that each message is delivered once and remains available until a
+consumer processes it and deletes it.
+
+AWS SQS queues can be configured to move messages that cannot be successfully
+processed by your application on a secondary queue, known as a dead-letter
+queue, letting you isolate problematic messages to determine why their
+processing did not succeed.
+
+#### Provision a standard AWS SQS queue
+
+Use a standard queue when the order of the items on your queue is not
+important and when your application can tolerate occasional duplication of
+messages.
+
+Run the following in the command line to provision a standard AWS SQS queue:
+
+```
+cf create-service aws-sqs-queue standard SERVICE_NAME
+```
+
+It will take between 1 and 5 minutes to set up the service instance. To check
+its progress run:
+
+```
+cf service SERVICE_NAME
+```
+
+where `SERVICE_NAME` is a unique descriptive name for this SQS queue.
+
+#### Provision a FIFO AWS SQS queue
+
+Use a FIFO (first-in-first-out) queue when the order of operations and events
+is critical, or where duplicates can't be tolerated.
+
+Run the following in the command line to provision a FIFO AWS SQS queue:
+
+```
+cf create-service aws-sqs-queue fifo SERVICE_NAME
+```
+
+It will take between 1 and 5 minuites to set up the service instance. To check
+its progress run:
+
+```
+cf service SERVICE_NAME
+```
+
+where `SERVICE_NAME` is a unique descriptive name for this SQS queue.
+
+### Bind an AWS SQS queue to your app
+
+You must bind the SQS queue to your app so you can get credentials to use the
+[AWS SQS API](https://docs.aws.amazon.com/cli/latest/reference/sqs/) to read
+from and write to the queue.
+
+You can bind an SQS queue using the command line or the manifest file.
+
+Run the following to bind an SQS queue to your app:
+
+```
+cf bind-service APP_NAME SERVICE_NAME
+```
+
+It will take between 1 and 5 minutes to set up the binding. To check its progress, run:
+
+```
+cf service SERVICE_NAME
+```
+
+where:
+
+- `APP_NAME` is the name of a deployed instance of your app, as specified in your app's manifest or push command
+- `SERVICE_NAME` is a unique descriptive name for this SQS queue
+
+When binding an SQS service to your app, you have the option of restricting the
+queue permissions given to that app. You can control this using `bind-service`'s `-c`
+flag to set the bind parameters `access_policy` JSON key, for example:
+
+```
+cf bind-service APP_NAME SERVICE_NAME -c '{"access_policy": "consumer"}'
+```
+
+The currently supported `access_policy` values are:
+
+- `full`, the default, full access permissions
```

Comment:
these bullet points need to follow on grammatically from the lead-in sentence, so need some tweaking